metadata-models/src/main/pegasus/com/linkedin/structured/StructuredPropertyDefinition.pdl

```diff
@@ -21,7 +22,11 @@ record StructuredPropertyDefinition {
     /**
     * The display name of the property. This is the name that will be shown in the UI and can be used to look up the property id.
     */
-    @Searchable = {}
+    @Searchable = {
+      "enableAutocomplete": true,
+      "fieldType": "WORD_GRAM",
```

Comment:
hmm we do want `WORD_GRAM` here? maybe `TEXT_PARTIAL` is a better option?

or looking at our docs within the last week it looks like we made this deprecated: https://docs.datahub.com/docs/metadata-modeling/extending-the-metadata-model/#migration-guide-for-deprecated-features

maybe it should just be `TEXT` - is this why we're not getting matches with less than 3 characters?